includes/class-wc-gateway-ppec-admin-handler.php

```diff
@@ -162,6 +162,7 @@ public function capture_payment( $order_id ) {
 
 				$params['AUTHORIZATIONID'] = $trans_id;
 				$params['AMT'] = floatval( $order_total );
+				$params['CURRENCYCODE'] = get_woocommerce_currency();
```

Comment:
I think this should be something like `$order->get_currency()`

Reply:
@allendav  it sounds much better, as it order currency in context. Thanks!

Reply:
If you can push an update to this PR today with that change, that would be awesome. If not, I can make the change and commit it.